quick-sort.js

```diff
@@ -0,0 +1,29 @@
+// Quick Sort
+
+function quickSort(array) {
+  if (array.length <= 1) {
+    return array;
+  }
+
+  if (array.length >= 1) {
```

Comment:
This `if` is unnecessary. If the previous `if` didn't work, it's obvious that the length is greater than 1.